coding/javascript.md

```diff
@@ -29,6 +29,11 @@ e.g. `<a data-js="index-link" href="/index">Index</a>`.
 
 ## Tooling
 
+### Build tools
+[Webpack](https://webpack.js.org/) is the preferred build tool for applications, and [Rollup](https://rollupjs.org/guide/en) is preferred for libraries.
+
+Rollup and Webpack for the most part have feature parity (Webpack has support for scope hoisting, and Rollup supports code splitting), so this is mostly a matter of taste and convenience, given Rollup is easier to configure for library builds.
```

Comment:
```
https://github.com/webpack/webpack/tree/master/examples/scope-hoisting
https://webpack.js.org/guides/code-splitting/
https://www.infoq.com/news/2019/03/rollup-code-splitting
https://medium.com/adobetech/optimizing-javascript-through-scope-hoisting-2259ef7f5994#ff06
```
I think we can remove the sentence that one has support and the other does not. They seem to be on par

Reply:
I think I edited too heavily, it used to read "Webpack *now* has support for scope hoisting"...